Flutter SDK docs update install instructions (#548)
amplitude/amplitude-docs: content/collections/flutter_sdk/en/flutter-sdk-4.md

## Install the SDK

> Run the following to grab the latest version of `amplitude_flutter`, which will add it to your `pubspec.yaml` then retrieve dependencies:

Reviewer: ```suggestion
Run the following command to get the latest version of `amplitude_flutter`, add it to your `pubspec.yaml`, then retrieve dependencies:
```
